website/content/contributors/_index.md

```diff
@@ -10,51 +10,60 @@ Welcome! Are you interested in contributing to one of CNCF hosted projects? This
 
 CNCF offers multiple ways to start contributing to the CNCF ecosystem, including either foundation-wide and project-wide opportunities.
 
+Though, contributing to the CNCF ecosystem is not just about coding. There are many other ways to contribute to the CNCF ecosystem, including writing documentation, creating tutorials, bringing ideas to meetings, organizing meetups, and more.
 
-## TOC
+## Open Source Beginners
 
-The CNCF TOC is the technical governing body of the CNCF Foundation. The detailed information on CNCF TOC, including its duties and responsibilities, together with the information on collaboration is listed on [CNCF TOC repo](https://github.com/cncf/toc/).
+Are you new to open source? If so, we recommend checking out our comprehensive guide, ["Start Contributing to Open Source"](getting-started.md). This guide is filled with helpful tips and tricks to help you get started with contributing to open source projects. You'll learn about communities and projects, how to find them, how to conform to community standards, and much more.
 
-## Technical Advisory Groups
+## Contribution Opportunities
 
-CNCF TAGs oversee and coordinate the interests pertaining to a logical area of needs of end users and/or projects. More details about the CNCF TAGs is available [here](https://github.com/cncf/toc/tree/main/tags).
+### Mentorship Programs
 
-_Note: CNCF TAGs were previously named Special Interest Groups (SIGs). Renaming SIGs was discussed in [this GitHub Issue](https://github.com/cncf/toc/issues/549)._
+The Cloud Native Computing Foundation participates in various mentoring programs, including:
 
-## Working Groups
+- [LFX Mentorship](https://github.com/cncf/mentoring/tree/master/lfx-mentorship) (previously known as Community Bridge) by the Linux Foundation
+- [Google Summer of Code](https://github.com/cncf/mentoring/tree/master/summerofcode) (GSoC)
+- [Google Season of Docs](https://github.com/cncf/mentoring/tree/master/seasonofdocs) (GSoD)
+- [Outreachy](https://github.com/cncf/mentoring/tree/master/outreachy)
```

Comment:
These links all are broken, did the pages move?

Reply:
Hmm, right.

I actually just move this content from the bottom of the page, but haven't actually checked the links.

Good catch.

Gonna fix it

Reply:
Copy&pasted the same content from Mentors WG document now.